src/database/migrations/20220417175812-modify_users_add_new_fields.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.addColumn('Users', 'age', {
+        type: Sequelize.INTEGER,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'gender', {
+        type: Sequelize.ENUM,
+        values: ['male', 'female', 'prefer not say'],
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'dob', {
+        type: Sequelize.DATEONLY,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'nationality', {
+        type: Sequelize.STRING,
+        defaultValue: 'rwandan',
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'department', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'preferredLanguage', {
+        type: Sequelize.STRING,
+      }),
+      queryInterface.addColumn('Users', 'preferredCurrency', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'address', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'lineManager', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'phoneNumber', {
+        type: Sequelize.STRING,
+      }),
+      queryInterface.addColumn('Users', 'maritalStatus', {
+        type: Sequelize.ENUM,
+        values: ['Single', 'Married', 'Widowed'],
+        defaultValue: 'Single',
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'isComplete', {
+        type: Sequelize.BOOLEAN(),
+        defaultValue: false,
+      }),
+    ]);
+  },
+  async down(queryInterface) {
+    // logic for reverting the changess
+    return Promise.all([
+      queryInterface.removeColumn('Users', 'age'),
+      queryInterface.removeColumn('Users', 'gender'),
+      queryInterface.removeColumn('Users', 'dob'),
```

Comment:
Does this dob means data of birth? I think you don't need to have column of age and column of dob. You can remove age since we can know it from date of birth

Reply:
@Procech20  age can be filled automatically depending on the dob entered